libs/sdk-backend-mockingbird/src/recordedBackend/index.ts

```diff
@@ -193,6 +193,7 @@ function recordedOrganization(organizationId: string, implConfig: RecordedBacken
         organizationId,
         securitySettings(): ISecuritySettingsService {
             return {
+                scope: `/gdc/domains/${organizationId}`, // TODO PR Question: is this alright for mocked backend to mirror bear one? Should the same functionality be reused from shared export?
```

Comment:
if the 'scope' is private prop of the security settings service then it does not matter. if it is visible from outside, then it may be better add a new impl config prop. this will contain function which transforms organization into scope. have the bear-like transformer as a default.

Reply:
It is used inside of the bear service impl as this information is sent to backend API. I also need it to access it in cached backend for caching entry creation, therefore it should be public as it is. I will make a transformer fn for it in mockingbird config 👍